cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -148,6 +153,5 @@ def update_sample_read_count(sample_id: str, q30_threshold: int, store: Store) -
             f"Updating sample {sample_id} with read count {sample_read_count} and setting sequenced at."
         )
         sample.reads = sample_read_count
-        sample.reads_updated_at = datetime.datetime.now()
-    else:
+        sample.reads_updated_at = sequenced_at
```

Comment:
I think this field only makes sense if it is the latest sequencing date ie do not overwrite a more recent date. Maybe it would make sense to rename to column to `last_sequenced_at`

Reply:
I agree, I'll add a check for that and rename the field.

Reply:
Could we not just revert the PRs which changed this from "sequenced_at" from the beginning? Or would there be any change apart from "sequenced_at" being changed to "last_sequenced_at"? Or is the parsing of the directory name what makes the behaviour more in line with when it was sequenced?

Reply:
Not sure what you mean @islean. I think the naming of the column is secondary here. The more important change is that `reads_updated_at` should not be the time when the post-processing is run.

Reply:
Nevermind, I think I am not adding anything beneficial to the discussion! Just thought that we are approaching how it was before some changes a month or two back. But I don't think that should impact any decisions taken now.

Reply:
Yeah, I think we have bounced a bit back and forth but this should hopefully be the last adjustment 🤞